lib/active_shipping/carriers/ups.rb

```diff
@@ -251,18 +251,29 @@ def build_rate_request(origin, destination, packages, options = {})
     # * shipper: who is sending the package and where it should be returned
     #     if it is undeliverable.
     # * ship_from: where the package is picked up.
-    # * service_code: default to '14'
-    # * service_descriptor: default to 'Next Day Air Early AM'
+    # * service_code: default to '03'
```

Comment:
UPS Ground seemed a far more reasonable default to me than Next Day Early AM, which is wildly expensive and probably only used in emergencies. Usually UPS Ground would be the default shipping option if you were ordering from a website that was shipping via ups and I imagine would be the default in most other situations as well. I'm assuming the reason it was 14 is because that's what is used in the example in UPS's documentation.